### PR TITLE
Dataproc GCS sample plus doc touchups

### PR DESCRIPTION
**Pull request by waprin**

### CODE REVIEW

File: dataproc/README.md [hunk 1]
```diff
@@ -20,49 +20,78 @@ Go to the [Google Cloud Console](https://console.cloud.google.com).
...
+
 To install, run the following commands. If you want to use  [virtualenv](https://virtualenv.readthedocs.org/en/latest/)
 (recommended), run the commands within a virtualenv.
 
     * pip install -r requirements.txt
 
 Create local credentials by running the following command and following the oauth2 flow:
 
-    gcloud beta auth application-default login
+    gcloud auth application-default login
```
Comment (jonparrott) [inline dataproc/README.md]: Our current auth guidance is to use service accounts.

File: dataproc/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ demonstrated here could also be accomplished using the Cloud Console or the gclo
 `list_clusters.py` is a simple command-line program to demonstrate connecting to the
```
Comment (jonparrott) [inline dataproc/README.md]: Meta: This readme is enormous. Is there a documentation page that it could link to instead?

Comment (waprin) [reply dataproc/README.md]: We do: https://cloud.google.com/dataproc/docs/tutorials/python-library-example . So yeah fair point, maybe I should cut this whole README, make sure all info's on that page?

Comment (jonparrott) [reply dataproc/README.md]: That would be strongly preferable.

Comment (waprin) [reply dataproc/README.md]: Ok, I pared down the README. I basically left all the information to actually run the samples, killed all the associated info and link to the tutorial at the top. I'll make a docs CL to add the note about reading from GCS with PySpark. LMK if that makes sense.

File: dataproc/README.md (hunk 1 above)
Comment (waprin) [reply dataproc/README.md]: Took this out, linked to our auth docs, noted we recommend service accounts.

**Comment by jonparrott** [comment]
LGTM, but travis is failing.

**Pull request merged**